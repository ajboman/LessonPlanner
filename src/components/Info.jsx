import React from 'react'

const Info = () => {
  return (
    <header className='w-full flex justify-center items-center flex-col'>
      <h1 className='head_text'>
        Plan Lessons Quick and Easy
      </h1>
      <h2 className='desc'>
        Provide as many details as you need. <br />
        More details will result in a more precise plan. <br />
        Save the Lesson Plan to view it in the Lessons page.
      </h2>
    </header>

  )
}

export default Info