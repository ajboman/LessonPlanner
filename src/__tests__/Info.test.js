import React from 'react';
import { render } from '@testing-library/react';
import Info from '../components/Info';

describe('Info component', () => {
  test('renders correctly', () => {
    const { container } = render(<Info />);
    expect(container).toMatchSnapshot();
  });

  test('displays the correct heading and description', () => {
    const { getByText } = render(<Info />);
    const headingElement = getByText('Plan Lessons Quick and Easy');
    const descriptionElement = getByText(
      'Provide as many details as you need. More details will result in a more precise plan. Save the Lesson Plan to view it in the Lessons page.'
    );
    expect(headingElement).toBeInTheDocument();
    expect(descriptionElement).toBeInTheDocument();
  });
});
