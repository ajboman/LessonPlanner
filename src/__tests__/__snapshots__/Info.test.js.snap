// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info component renders correctly 1`] = `
<div>
  <header
    class="w-full flex flex-col items-center justify-center py-12 px-6 text-center bg-gradient-to-r from-secondary to-primary text-text rounded-lg shadow-xl"
  >
    <h1
      class="text-5xl font-bold mb-6"
    >
      Plan Lessons Quick and Easy
    </h1>
    <h2
      class="text-lg font-medium space-y-2"
    >
      Provide as many details as you need.
      <br />
      More details will result in a more precise plan.
      <br />
      Login to save the Lesson Plan and view it in the Lessons page.
      <br />
      Verify your email for longer responses.
      <br />
      Example will not affect your lesson.
    </h2>
  </header>
</div>
`;
